src/lightning/fabric/CHANGELOG.md

```diff
@@ -35,7 +35,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Added `Fabric.init()` context manager to instantiate tensors or models efficiently directly on device and dtype ([#17488](https://github.com/Lightning-AI/lightning/pull/17488))
 
-- Added `Fabric.init_module()` context manager to instantiate large models efficiently directly on device, dtype, and sharding them ([#17462](https://github.com/Lightning-AI/lightning/pull/17462))
```

Comment:
Maybe "with sharded support" would be more appropriate instead of removing this

Reply:
udpated